build_rpm.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -xe
+#
+# Build from master by default.
+# $1 argument can be used to specify another ref (i.e. user-dev).
+#
+
+GITREF=${1:-master}
+COMMIT_ID=$(git show-ref --hash refs/heads/$GITREF)
+SHORT_COMMIT_ID=$(git show-ref --hash=7 refs/heads/$GITREF)
+TAR_FILE=joy-$SHORT_COMMIT_ID.tar
+TGZ_FILE=joy-$SHORT_COMMIT_ID.tar.gz
+
+git archive --format=tar --prefix="joy/" $COMMIT_ID > $TAR_FILE
+if [ "$?" != 0 ]; then
+    echo Failed to create git archive tar
+    exit 1
+fi
+#if [ "$GITREF" != "master" ]; then
```

Comment:
Do you need this code that was commented out? Might have forgotten to remove it.